tools/Crank/run-benchmarks.ps1

```diff
@@ -64,27 +64,28 @@ $crankConfigPath = Join-Path `
 
 $isLinuxApp = $CrankAgentVm -match '\blinux\b'
 
-$functionAppRootPath = if ($isLinuxApp) { "/home/$UserName/FunctionApps" } else { 'C:\FunctionApps' }
-$functionAppPath = Join-Path `
-                    -Path $functionAppRootPath `
-                    -ChildPath $FunctionApp
+$functionAppPath = if ($isLinuxApp) {
+                       "/home/$UserName/FunctionApps/$FunctionApp"
+                   } else {
+                       "C:\FunctionApps\$FunctionApp"
+                   }
 
 $tmpPath = if ($isLinuxApp) { "/tmp" } else { 'C:\Temp' }
 $tmpLogPath = if ($isLinuxApp) { "/tmp/functions/log" } else { 'C:\Temp\Functions\Log' }
 
 if ($UseHttps) {
     $aspNetUrls = "http://localhost:5000;https://localhost:5001"
-    $profile = "localHttps"
+    $profileName = "localHttps"
```

Comment:
`$profile` is a built-in variable in PowerShell, so renaming this one just in case.